lib/mini_portile2/mini_portile_cmake.rb

```diff
@@ -48,4 +48,16 @@ def make_cmd
   def cmake_cmd
     (ENV["CMAKE"] || @cmake_command || "cmake").dup
   end
+
+  private
+
+  def generator_available?(generator_type)
+    action = 'cmake_help'
+    execute(action, [cmake_cmd, '--help'])
+    filename = log_file(action)
```

Comment:
Could we be using `IO.popen` or `Open3` to capture stdout/stderr directly, instead of running it through a log file?

Reply:
@flavorjones We could. I was just trying to use the existing `execute` method. Would you prefer that?

Reply:
Changed to `Open3.capture2`.